sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/WebAppMsiHandler.java

```diff
@@ -78,7 +78,7 @@
             return this;
         } else if (siteInner.identity().type().equals(ManagedServiceIdentityType.SYSTEM_ASSIGNED)) {
             siteInner.identity().withType(ManagedServiceIdentityType.NONE);
-        } else if (siteInner.identity().type().equals(ManagedServiceIdentityType.SYSTEM_ASSIGNED__USER_ASSIGNED)) {
+        } else if (siteInner.identity().type().equals(ManagedServiceIdentityType.SYSTEM_ASSIGNED_USER_ASSIGNED)) {
```

Comment:
Cause is that autorest.java changed the logic on enum value name. So try regen the code. Then found that the new logic is really worse than before, hence revert that logic. Only fix this one by autorest.java.